### ISSUE TITLE
[BUG] tracer can send duplicate packets in HEP

### ISSUE THREAD

**Issue by jes** [OpenSIPS/opensips#3106]
**OpenSIPS version you are running**

```
3.4.0-dev from  028704c6004a227054dc8dab3bc14c4b263c1df6
```

**Describe the bug**
We have config like:

```
...
loadmodule "tm.so"
loadmodule "proto_hep.so"
modparam("proto_hep", "hep_id", "[hid]127.0.0.1:10000; transport=udp; version=3")
modparam("proto_hep", "hep_id", "[hid2]127.0.0.1:10001; transport=udp; version=3")
socket=hep_udp:0.0.0.0:9999
loadmodule "tracer.so"
modparam("tracer", "trace_on", 1)
modparam("tracer", "trace_id", "[tid]uri=hep:hid")
modparam("tracer", "trace_id", "[tid2]uri=hep:hid2")
...

route {
...
trace("tid", "t", "sip");
trace("tid2", "t", "sip");
...
}
```
And we sometimes see duplicate HEP packets coming into tid/tid2.

**To Reproduce**
1. Load tm, proto_hep, tracer
2. Set up 2 HEP endpoints
3. trace(..., "t", "sip") to both endpoints
4. Using sngrep or similar, observe that you sometimes get duplicate HEP coming out

**Expected behavior**
I expected OpenSIPS to send each packet to each trace_id only once.

**Relevant System Logs**

**OS/environment information**
 - Operating System: Centos 7
 - OpenSIPS installation: git
 - other relevant information:

**Additional context**
You can work around the problem like so:

```
...
loadmodule "tm.so"
loadmodule "proto_hep.so"
modparam("proto_hep", "hep_id", "[hid]127.0.0.1:10000; transport=udp; version=3")
modparam("proto_hep", "hep_id", "[hid2]127.0.0.1:10001; transport=udp; version=3")
socket=hep_udp:0.0.0.0:9999
loadmodule "tracer.so"
modparam("tracer", "trace_on", 1)
modparam("tracer", "trace_id", "[tid]uri=hep:hid")
modparam("tracer", "trace_id", "[tid]uri=hep:hid2")
...

route {
...
trace("tid", "t", "sip");
...
}
```

i.e. make your one trace_id go to both hep_id's, instead of having a separate hep_id for each trace_id, which is what we're doing now. But the existing behaviour seems buggy to me so I'm reporting it regardless.

**Comment by razvancrainea** [comment]
This does seem to be duplicate of #3082 . Closing this one, please follow on the other ticket.
